tests/runtests.py

```diff
@@ -449,6 +449,7 @@ def test_cmd_keystone_assemble(self):
             res = gdb_start_silent_cmd(cmd)
             self.assertNoException(res)
             self.assertTrue(len(res.splitlines()) > 1)
+            self.assertNotIn("Undefined command", res)
```

Comment:
Probably better to replace the beginning of `test_cmd_keystone_assemble()` with something like:

```python
    def test_cmd_keystone_assemble(self):
        cmd = "gef missing"
        res = gdb_start_silent_cmd(cmd)
        self.assertNotIn("Command `assemble` is missing, reason", res)

        valid_cmds = [
        [...]
```
So the test is done only once.

This also applies to `test_cmd_capstone_disassemble()`